src/directory/directory.mjs

```diff
@@ -1455,10 +1455,10 @@ export const directory = {
           {
             title: 'Copy files',
             route: '/lib-v1/storage/copy',
-            filters: ['js', 'flutter']
```

Comment:
This page should not exist for flutter lib-v1, current docs: https://docs.amplify.aws/lib-v1/storage/copy/q/platform/flutter/